ontology/disdriv.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ontology_detail
+id: disdriv
+contact:
+  email: lynn.schriml@gmail.com
+  label: Lynn Schriml
+  github: lschriml
+description: Ontology for drivers and triggers of human diseases, built to classify ExO ontology exposure stressors. An application ontology. Built in collaboration with EnvO, ExO, ECTO and ChEBI.
+twitter: diseaseontology
+domain: disease
+homepage: http://www.disease-ontology.org
+products:
+  - id: disdriv.owl
+browsers:
+  - label: DO
+    title: DO Browser
+    url: http://www.disease-ontology.org/
+taxon:
+  id: NCBITaxon:9606
+  label: Homo sapiens
+title: Disease Drivers Ontology
+build:
+  source_url: https://github.com/DiseaseOntology/DiseaseDriversOntology/tree/main/src/ontology/disdriv.owl
+tracker: https://github.com/DiseaseOntology/DiseaseDriversOntology/issues
+license:
+  url: https://creativecommons.org/publicdomain/zero/1.0/
+  label: CC0 1.0
+usages:
```

Comment:
I would prefer if we remove the usage, until there is some actual usage..

Reply:
I would say that we are using "environmental" very broadly here to include things like diet and socioeconomics. The issue then becomes our reliance on other ontologies to build ECTO classes because there is no socioeconomics ontology.